Essentials/src/messages.properties

```diff
@@ -609,7 +609,9 @@ versionOutputUnsupportedPlugins=\u00a76You are running \u00a7dunsupported plugin
 versionMismatch=\u00a74Version mismatch\! Please update {0} to the same version.
 versionMismatchAll=\u00a74Version mismatch\! Please update all Essentials jars to the same version.
 voiceSilenced=\u00a76Your voice has been silenced\!
+voiceSilencedNew=\u00a76Your voice has been silenced for {0}\!
```

Comment:
I'm not a fan of this, because it largely duplicates an existing tl key (which then is no longer used as far as I can tell). I think it would make much more sense to add the placeholders to the existing messages instead.